packages/simon/config/index.js

```diff
@@ -167,10 +167,16 @@ export default {
     buttonPlacketStyle: {
       list: ['classic', 'seamless'],
       dflt: 'classic',
+      hide: (options)  => {
+        return options.seperateButtonPlacket
+      }
```

Comment:
Perhaps is a coding style preference, but a fat arrow function that immediately returns does not need curly braces.
This is equivalent:
```js
hide: options => options.seperateButtonPlacket
```

Reply:
It wasn't my preference! I just didn't know. I like the one-liner, thanks!
Done.